src/package.xml

```diff
@@ -1,126 +1,130 @@
+    <column name="Alternative" displayName="Alternative Scenario ID" dataType="Integer"
+      validationType="WholeNumber" validationCondition="None" isOptional="False" allowDbNull="False" />
+  </dataSheet>
+
+  <dataSheet name="outputSpatialOverall" displayName="Overall" isSingleRow="True">
+    <column name="overallCurrentDifferenceRaster" displayName="Normalized current difference"
+      dataType="String" isExternalFile="True"
+      externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*" isRaster="True" />
+    <column name="overallDifferenceRaster" displayName="Cross-category difference" dataType="String"
+      isExternalFile="True" externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*"
+      isRaster="True" />
+  </dataSheet>
+
+  <dataSheet name="outputSpatialCategory" displayName="Per Category">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="perCategoryRaster" displayName="File path" dataType="String" isExternalFile="True"
+      externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*" isRaster="True" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularDifferences" displayName="Differences Summary">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="amountAreaDifference" displayName="Area difference" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+    <column name="percentCoverDifference" displayName="Proportion difference" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularChange" displayName="Transitions Summary">
+    <column name="movementTypesIDfrom" displayName="From" dataType="String" />
+    <column name="movementTypesIDto" displayName="To" dataType="String" />
+    <column name="amoutAreaChange" displayName="Area change" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+    <column name="percentCoverChange" displayName="Proportion change" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularJaccard" displayName="Jaccard Dissimilarity">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="dissimilarity" displayName="Dissimilarity" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <!--Transformers-->
+
+  <transformer
+    name="omniscapeImpactTransformer"
+    displayName="3 - Connectivity Impact Assessment"
+    transformerSpecificOutput="False"
+    isRunnable="True"
```

Comment:
This element no longer exists and can be removed

Reply:
Just noticed that the transformerSpecificOutput element is in the omniscape 2.1.1 
package.xml